### PR TITLE
Add columns to SHOW DATABASES for autonomous clusters

### PR DESCRIPTION
**Pull request by mnd999**
Add additional columns to SHOW DATABASES to support autonomous
clustering feature.

Depends on https://github.com/neo-technology/neo4j/pull/16522

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 1]
```diff
@@ -754,14 +754,22 @@ label:updated[]
 SHOW DATABASES YIELD *
 ----
 a|
-New columns have been added to the result:
+Changes have been made to the default columns in the result:
+
+* The `writer` columns has been added.
+* The values returned in the `role` column have changes to be just `primary` and `secondary` or `unknown`
+* The `error` column has been renamed to `statusMessage`
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]:
```suggestion
* The values returned in the `role` column have changes to be just `primary` and `secondary` or `unknown`.
* The `error` column has been renamed to `statusMessage`.
```

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 2]
```diff
@@ -754,14 +754,22 @@ label:updated[]
 SHOW DATABASES YIELD *
```
Comment (Hunterness) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: should we add/update so this isn't just with YIELD?

**Review by gfx54b — APPROVED** [review]

**Review by Hunterness — APPROVED** [review]

**Review by Hunterness — APPROVED** [review]

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc (hunk 2 above)
Comment (mnd999) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: I removed the YIELD *

File: cypher/cypher-docs/src/docs/dev/ql/administration/show-databases-columns.asciidoc [hunk 3]
```diff
@@ -7,12 +7,17 @@
 |databaseID|The database unique ID.||{check-mark}|
 |serverID|The server instance ID.||{check-mark}|
 |address|Instance address in a clustered DBMS. The default for a standalone database is `neo4j://localhost:7687`. |{check-mark}|{check-mark}|
-|role|The current role of the database (`standalone`, `leader`, `follower`, `read_replica`, `unknown`).|{check-mark}|{check-mark}|
+|role|The current role of the database (`primary`, `secondary`, `unknown`).|{check-mark}|{check-mark}|
+|writer|This will be true if this database node accepts writes which will be the case if this node is the leader for this database in a cluster or if this is a standalone instance.|{check-mark}|{check-mark}|
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/ql/administration/show-databases-columns.asciidoc]:
```suggestion
|writer| `true` for the database node that accepts writes (this node is the leader for this database in a cluster or  this is a standalone instance.|{check-mark}|{check-mark}|
```

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 4]
```diff
@@ -751,17 +751,25 @@ label:functionality[]
 label:updated[]
 [source, cypher, role="noheader"]
 ----
-SHOW DATABASES YIELD *
+SHOW DATABASES
 ----
 a|
-New columns have been added to the result:
+Changes have been made to the default columns in the result:
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]:
```suggestion
Changes to the default columns in the result:
```

**Review by renetapopova — APPROVED** [review]
Thanks, @mnd999! Sorry for the delay. I couldn't build it locally, but the tests in TC are green, so I guess everything should be fine. I have just a couple of editorial suggestions.

**Pull request merged**